src/api-explorer/v4-0/Expense.swagger2.json

```diff
@@ -0,0 +1,3701 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Manage expected expense entries attached to a Request document.",
+    "version": "4.0",
+    "title": "Workflow"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/api/v4",
+  "tags": [
+    {
+    "name": "Resources",
+    "description": " "
+    }
+  ],
+  "paths": {
+    "/travelrequest/requests/{requestUuid}/expenses": {
+      "get": {
+        "tags": [
+          "expense-resource"
+        ],
+        "summary": "Get expected expenses attached to a request",
+        "operationId": "listUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "Name of the perimeter defining the requests to get",
```

Comment:
It should be something like "Uuid of the Request holding the Expenses"
Check wording with Charlotte.